tests/activemq5-unit-tests/pom.xml

```diff
@@ -41,7 +41,7 @@
       <jasypt-version>1.9.3</jasypt-version>
       <directory-version>2.0.0-M6</directory-version>
       <activeio-core-version>3.1.4</activeio-core-version>
-      <byteman.version>2.2.1</byteman.version>
+      <byteman.version>4.0.18</byteman.version>
```

Comment:
There is another optional module using byteman, with its own copy of this property, set to a different 4.0.x version. They should probably be made consistent e.g move property up a pom so they share, or have these tests moved to that module and this dropped.